### PR TITLE
Update Link Checker action

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -16,3 +16,10 @@ updates:
   commit-message:
     prefix: patch
   rebase-strategy: disabled
+- package-ecosystem: "github-actions"
+  directory: ".github/workflows"
```
Comment (klutchell) [inline .github/dependabot.yml:20]:
```suggestion
  directory: "/"
```
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot

Comment (vipulgupta2048) [reply .github/dependabot.yml:20]: I wonder why they don't let you go the other way around of specifying the correct directory.

**Comment by klutchell** [comment]
Squash this to one commit and I'll approve!

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**